pkg/syncer/metadatasyncer.go

```diff
@@ -408,6 +408,31 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		}()
 	}
 
+	// Trigger get pv to vsan/vvol mapping on vanilla cluster
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
+		pvToVvolMappingTicker := time.NewTicker(time.Duration(getVolumeHealthIntervalInMin(ctx)) * time.Minute)
```

Comment:
Move the FSS check here. Basically, if FSS is not enabled, none of this piece of code should be executed at all.